### DISCUSSION TITLE
I would like to know how these cards are implemented under Winui3?

### DISCUSSION THREAD

**Discussion by Melendez1209** [microsoft/microsoft-ui-xaml#8245] [Q&A] [answered]
I would like to know how these cards are implemented under Winui3?
![image](https://user-images.githubusercontent.com/92637479/222717302-6c44debf-e842-4e8e-b134-b7d3b1621670.png)

**Comment by castorix** [comment, marked as answer]
You can see the source code :  https://github.com/microsoft/WinUI-Gallery/tree/main/WinUIGallery/Controls
It is the _HeaderTile_ control

**Reply by Melendez1209** [reply]
What are the necessary parameters I should pass in when using it?

**Reply by Melendez1209** [reply]
![image](https://user-images.githubusercontent.com/92637479/222870455-392a6108-0e4a-432c-95b7-d797a872cd9a.png)
Why does this look odd?

**Reply by castorix** [reply]
> What are the necessary parameters I should pass in when using it?

You can find the XAML pages where it is used, like NewControlsPage.xaml or TileGallery.xaml
